tests/settings.py

```diff
@@ -1,3 +1,4 @@
 from ephios.settings import *  # no-qa
 
 LANGUAGE_CODE = "en"
+SITE_URL = "http://testserver"
```

Comment:
shouldn't this overwrite `GET_SITE_URL`?

Reply:
I experimented a lot with how the live test server behaves esp. in conjunction with changes to the settings and it all seemed a bit messy. Not sure if this is still needed anyways, tests are still a big WIP ^^ (or rather WIB)

Reply:
work in bahn? work in bitterness?

Reply:
work in backlog, but your options fit as well ^^